com.github.ahrm.sioyek.yaml

```diff
@@ -0,0 +1,75 @@
+---
+app-id: com.github.ahrm.sioyek
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: sioyek
+
+finish-args:
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --filesystem=/tmp
+  - --share=ipc
+  # Sioyek fails with wayland (invisible window)
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --device=shm
+  - --device=dri
+
+modules:
+  - org.boost.Boost.json
+  - qtbase-5-12.yml
+  - name: qt5-wayland
+    buildsystem: qmake
+    cleanup:
+      - /bin
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/*.prl
+      - /lib/cmake
+      - /lib/mkspecs
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: http://qt.mirror.constant.com/archive/qt/5.12/5.12.12/submodules/qtwayland-everywhere-src-5.12.12.tar.xz
+        sha256: 47d074357d6b3e8bf2873462cad85555f9dc2899aa8422bbcc81ca31d4fd5b27
+      - type: patch
+        path: patches/qt/qtwayland-use-gnome-platform-theme-on-gnome-based-desktops.patch
+      - type: patch
+        path: patches/qt/Fix-installing-qtwayland-without-qtquick-or-opengl-s.patch
+  - "shared-modules/glu/glu-9.json"
+  - name: sioyek
+    buildsystem: simple
+    build-commands:
+      - ./build_linux.sh
+      - mv build /app/sioyek
+      - ln -s /app/sioyek/sioyek /app/bin/sioyek
+    post-install:
+      - install -Dm644 sioyek.png /app/share/icons/hicolor/256x256/apps/com.github.ahrm.sioyek.png
+      - desktop-file-install --delete-original --dir=/app/share/applications com.github.ahrm.sioyek.desktop
+      - install -Dm644 -t /app/share/metainfo com.github.ahrm.sioyek.metainfo.xml
+    sources:
+      - type: git
+        url: git://github.com/ahrm/sioyek.git
+        commit: 5c325e7bb340aeb55ec4a4b5678b6cf91bc0ac97
+        tag: 'v1.1.0'
+      - type: patch
+        path: patches/sioyek_build_fix.patch
+      - type: file
+        url: https://github.com/ahrm/sioyek/raw/v1.1.0/resources/sioyek-icon-linux.png
```

Comment:
you don't need this. Just take it from the tree checkout above.

Reply:
Done.